app/controllers/api/v1/servers_controller.rb

```diff
@@ -0,0 +1,214 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ServersController < ApplicationController
+      include ApiHelper
+
+      before_action :set_server, only: [:update, :destroy, :panic]
+
+      # Return a list of the configured BigBlueButton servers
+      # GET /bigbluebutton/api/v1/servers
+      #
+      # Successful response:
+      # [
+      #   {
+      #     "id": String,
+      #     "url": String,
+      #     "secret": String,
+      #     "state": String,
+      #     "load": String,
+      #     "load_multiplier": String,
+      #     "online": String
+      #   },
+      #   ...
+      # ]
+      def index
+        servers = Server.all
+
+        if servers.empty?
+          render json: { message: 'No servers are configured' }, status: :ok
+        else
+          server_list = servers.map do |server|
+            {
+              id: server.id,
+              url: server.url,
+              secret: server.secret,
+              state: if server.state.present?
+                       server.state
+                     else
+                       server.enabled ? 'enabled' : 'disabled'
+                     end,
+              load: server.load.presence || 'unavailable',
+              load_multiplier: server.load_multiplier.nil? ? 1.0 : server.load_multiplier.to_d,
+              online: server.online ? 'online' : 'offline'
+            }
+          end
+
+          render json: server_list, status: :ok
+        end
+      end
+
+      # Add a new BigBlueButton server (it will be added disabled)
+      # POST /bigbluebutton/api/v1/servers
+      #
+      # Expected params:
+      # {
+      #   "server": {
+      #     "url": String,                 # Required: URL of the BigBlueButton server
+      #     "secret": String,              # Required: Secret key of the BigBlueButton server
+      #     "load_multiplier": Float       # Optional: A non-zero number, defaults to 1.0 if not provided or zero
+      #   }
+      # }
+      def create
+        if server_create_params[:url].blank? || server_create_params[:secret].blank?
+          render json: { message: 'Error: Please input at least a URL and a secret!' }, status: :unprocessable_entity
+        else
+          tmp_load_multiplier = server_create_params[:load_multiplier].present? ? server_create_params[:load_multiplier].to_d : 1.0
```

Comment:
Can you not do `tmp_load_mutltiplier = server_create_params[:load_multiplier].presence&.to_d || 1.0` or something like that?